app/main.py

```
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) < 3 or command.split()[0] != "cp":
+        return
+    old_file = command.split()[1]
+    new_file = command.split()[2]
+    if old_file == new_file:
+        return
+    try:
+        with open(old_file, "r") as old, open(new_file, "w") as new:
+            for line in old.readlines():
+                new.write(line)
+    except FileNotFoundError:
+        print(f"No such file or directory: '{old_file}'")
```

This violates the checklist Clean Code guidance: "Add comments, prints, and functions to check your solution when you write your code. Don't forget to delete them when you are ready to commit and push your code." 

Details: the `print(f"No such file or directory: '{old_file}'")` in the exception handler should be removed before final submission. The task/tests typically expect functions to not print extraneous messages; either let the exception propagate or handle it silently per the task's expectations.